arrow/parser.py

```diff
@@ -49,6 +50,7 @@ class DateTimeParser(object):
     _TIMESTAMP_RE = re.compile(r"^\-?\d+\.?\d+$")
     _TIMESTAMP_EXPANDED_RE = re.compile(r"^\-?\d+$")
     _TIME_RE = re.compile(r"^(\d{2})(?:\:?(\d{2}))?(?:\:?(\d{2}))?(?:([\.\,])(\d+))?$")
+    _WEEK_DATE_RE = re.compile(r"^(\d{4})([\-])?W(\d{2})(?:([\-])?(\d))?$")
```

Comment:
We should remove the `^$` since this will not work with natural language strings otherwise.